src/pages/home/report/ReportActionsView.tsx

```diff
@@ -298,35 +265,15 @@ function arePropsEqual(oldProps, newProps) {
         return false;
     }
 
-    if (newProps.isSmallScreenWidth !== oldProps.isSmallScreenWidth) {
-        return false;
-    }
-
-    if (newProps.isComposerFullSize !== oldProps.isComposerFullSize) {
-        return false;
-    }
-
-    if (lodashGet(newProps, 'policy.avatar') !== lodashGet(oldProps, 'policy.avatar')) {
-        return false;
-    }
-
-    if (lodashGet(newProps, 'policy.name') !== lodashGet(oldProps, 'policy.name')) {
-        return false;
-    }
```

Comment:
In this component, policy was used to [pass it](https://github.com/Expensify/App/pull/35566/files#diff-96a3aa9dfe7b38cb9d5223d9d2244b63cd4defe8ebc21a8ebd5277e2104624c1L265) to `ReportActionsList`. But it wasn't used inside `ReportActionsList`.